src/onnx/onnx.cpp

```diff
@@ -52,7 +52,9 @@ namespace onnx = onnx_for_migraphx;
 
 struct onnx_parser
 {
-    using attribute_map = std::unordered_map<std::string, onnx::AttributeProto>;
+    using attribute_map   = std::unordered_map<std::string, onnx::AttributeProto>;
+    using nearest_op      = std::function<std::size_t(std::size_t, double)>;
+    using original_idx_op = std::function<double(std::size_t, std::size_t, std::size_t, double)>;
```

Comment:
Can these typedefs be moved into each corresponding function?

Reply:
fixed